x-pack/plugins/triggers_actions_ui/public/application/components/builtin_alert_types/threshold/index.ts

```diff
@@ -84,20 +89,32 @@ export function getAlertType(): AlertTypeModel {
           )
         );
       }
-      if (threshold && threshold.length > 0 && !threshold[0]) {
+      if (!threshold || threshold.length === 0 || (threshold.length === 1 && !threshold[0])) {
```

Comment:
this seems like it would fall into the if block and generate a message if `threshold` was `[0]`.  So I think the `!threshold[0]` is either not needed if we know it's a number, or if it could be null | undefined, change to `threshold[0] == null`.